src/iuse.cpp

```diff
@@ -453,17 +453,21 @@ void remove_radio_mod( item &it, Character &p )
     it.unset_flag( flag_RADIOCARITEM );
 }
 
-// Checks that the player does not have an active item with LITCIG flag.
-static bool check_litcig( player &u )
+// Checks that the player can smoke
+cata::optional<std::string> iuse::can_smoke( const player &u )
 {
     auto cigs = u.items_with( []( const item & it ) {
         return it.active && it.has_flag( flag_LITCIG );
     } );
-    if( cigs.empty() ) {
-        return true;
+
+    if( !cigs.empty() ) {
+        return _( "You're already smoking a %s!" ), cigs[0]->tname();
```

Comment:
```suggestion
        return string_format( _( "You're already smoking a %s!" ), cigs[0]->tname() );
```

Reply:
Doesn't _() do that already?

Reply:
I think you are returning a comma expression now.

Reply:
If it did, the arguments would need to be inside `_()`. I'm surprised nothing is complaining about this.